docs/examples/quick-start-with-terraform.md

```diff
@@ -0,0 +1,302 @@
+# Quick Start with Terraform
+
+This tutorial aims at presenting how to set up an environment with Liqo installed via Terraform and explore its most notable capabilities.
+You will create a _virtual cluster_ by peering two Kubernetes clusters and deploy a simple application on it.
+
+## Provision the infrastructure
+
+Before starting the tutorials below, you should ensure the following software is installed on your system:
+
+- [**Docker**](https://docker.io), the container runtime.
+- [**Kubectl**](https://kubernetes.io/docs/tasks/tools/install-kubectl/), the command-line tool for Kubernetes.
+- [**Terraform**](https://developer.hashicorp.com/terraform/tutorials/aws-get-started/install-cli), the command-line tool for Terraform.
+
+Then, let's open a terminal on your machine and launch the following script, which creates the complete infrastructure with all playground already set up.
+
+{{ env.config.html_context.generate_clone_example_tf('quick-start-terraform') }}
+
+## Analyze the infrastructure and code
+
+With this command two KinD clusters have been created, peered with each other and a namespace has been offloaded on which two pods and a service have been deployed.
+
+### KinD resource
+
+Let's start from the creation of the KinD clusters: the resource in charge of building them is the `kind_cluster` resource of the provider [tehcyx](https://registry.terraform.io/providers/tehcyx/kind/latest/docs) which starting from configuration parameters (such as cluster_name, service_subnet/pod_subnet) generates the clusters and related config files needed by other providers to set up the infrastructure.
+
+You can inspect the deployed clusters by typing:
```

Comment:
Where do you have to type this command? On your workstation? In a specific cluster?

Reply:
Updated with correct phrase.